src/style.css

```diff
@@ -1,3 +1,117 @@
+html {
+  font-family: "Roboto", "Open Sans", sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  font-style: normal;
+  --main-accent: #060b35;
+  --secondary: #616070;
+  --white: #fff;
+  --blue: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  display: flex;
+  justify-content: center;
+}
+
+.image {
+  height: 134px;
+  width: 160px;
+}
+
+.card__title {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: var(--main-accent);
+  margin-top: 40px;
+}
+
+.card__code {
+  color: var(--secondary);
+  margin-top: 4px;
+}
+
+.card__rating {
+  display: flex;
+  align-items: flex-end;
+  justify-content: space-between;
+  margin-top: 16px;
+}
+
+.stars {
+  display: flex;
+  width: 96px;
+}
+
+.stars__star {
+  background-position: center;
+  background-repeat: no-repeat;
+  background-image: url(images/star.svg);
+  margin-left: 4px;
+  width: 16px;
+  height: 16px;
+}
+
+.stars :nth-child(-n + 1) {
+  margin: 0;
+}
+
+.stars :nth-child(-n + 4) {
+  background-image: url(images/star-active.svg);
+}
```

Comment:
<img width="830" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/98764641/b9d57070-475a-4e99-8df6-cb1dda04a9a1">
if you have questions about it, feel free to ask in the chat